compiler_setup/nvidia_acc.sh

```diff
@@ -0,0 +1,48 @@
+# Build settings for the NVIDIA compiler with OpenACC
+# ===================================================
+# Fortran compiler
+F90=nvfortran
+# C compiler
+CC=nvcc
+CFLAGS="-g"
+# Fortran compiler flags
+F90FLAGS="-O3 -Minfo=all"
```

Comment:
Please explain why -MInfo=all is used and what it does.

Reply:
Done.